test/ui/ChatterPostAttachmentTest.ts

```diff
@@ -24,10 +24,47 @@ export function ChatterPostAttachmentTest() {
           .getAttribute('href')
       ).toContain(ChatterUtils.buildURI(result.clickUri, result.raw.sffeeditemid, result.raw.sfcontentversionid));
     });
-
+    it('should use sfcontentfilename if present', () => {
```

Comment:
These tests contains quite a bit of repetition/copy paste.

You could extract utilities function inside a `describe` scope.